charts/app-config/values-production.yaml

```
@@ -190,10 +190,10 @@ govukApplications:
     appResources:
       limits:
         cpu: 4
-        memory: 1500Mi
+        memory: 2000Mi
       requests:
         cpu: 2
-        memory: 1Gi
+        memory: 1500mi
```

This needs to be big M otherwise you're requesting 1500 millibytes.
```suggestion
        memory: 1500Mi
```